builders/pallets-precompiles/precompiles/gmp.md

```diff
@@ -0,0 +1,155 @@
+---
+title: GMP Precompile
+description: Learn about the GMP precompile on Moonbeam and how to use it with the Moonbeam Routed Liquidity program provided by bridges like Wormhole.
+keywords: solidity, ethereum, GMP, wormhole, moonbeam, bridge, connected, contracts, MRL
+---
+
+# Interacting with the GMP Precompile
+
+![GMP Precompile Moonbeam Banner](/images/builders/pallets-precompiles/precompiles/gmp/gmp-banner.png)
+
+## Introduction {: #introduction } 
```

Comment:
I think this introduction needs to be a bit more informative. Like what does GMP stand for? Maybe here is the best place to mention it's only available on Wormhole. 

Imagine if you had no prior knowledge and you stumbled upon this page - would you understand what it's about? I don't think you have to go into details on everything but including links saying you should be familiar with Wormhole or if you want to find more info out about XCM, check out the XCM docs. I see you mentioned briefly cross-chain protocols, that's a good spot to add a link to the protocols index page that we have